cloud/linode/loadbalancers.go

```diff
@@ -450,7 +450,15 @@ func (l *loadbalancers) getNodeBalancerByHostname(ctx context.Context, service *
 }
 
 func (l *loadbalancers) getNodeBalancerByIPv4(ctx context.Context, service *v1.Service, ipv4 string) (*linodego.NodeBalancer, error) {
-	lbs, err := l.client.ListNodeBalancers(ctx, nil)
+	// TODO: use this constructor once we merge in linodego v1 update
```

Comment:
Agree. Though we have a bit of a chicken-egg problem - we don't have v1 committed here, but the v1 PR doesn't have NB filtering (so it'd conflict if we put it there). To avoid the rebase issues, I'll just move this code block to the PR itself and once this is merged, we'll rebase and commit the new filtering logic in that PR.

Reply:
Thank you